src/Traits/QueryComponentTraits/QueryComponentFilterTrait.php

```diff
@@ -33,15 +33,31 @@ trait QueryComponentFilterTrait
      */
     protected $clientQuery;
 
+    /**
+     * Convert a field/value filter pair to a Criteria object that can build part of a Solr query
+     *
+     * @param string $field
+     * @param mixed $value
+     * @return Criteria
+     */
+    protected function convertFilterToCriteria(string $field, $value): Criteria
```

Comment:
Missing full unit tests

Reply:
I'm not exactly happy with the method name, "my convention" would rather make it `buildCriteria`

Reply:
** `buildCriteriaFilter`